### DISCUSSION TITLE
r/maxus-V80-RV-unexpected-fuel-efficiency

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#51] [Announcements]
# r/maxus-V80-RV-unexpected-fuel-efficiency

房车用了两年多，这个油耗我是没有想到的

https://blog.ops-coffee.cn/r/maxus-V80-RV-unexpected-fuel-efficiency

**Comment by tjsky** [comment]
开CRV综合油耗都在百公里9升多的话，房车整体油耗在8.5L/百公里左右是很低了呀，柴油还便宜。我之前的CR-V综合油耗只有接近8，如果快速路跑的多了时期，还最低见过7.2的油耗。

**Reply by ops-coffee** [reply]
CRV主要通勤开，通勤太堵了，不是很堵的话8左右，正常的

**Comment by yaochao** [comment]
写的真好，点个赞
